data/scenarios/02Tutorials/06-bind.yaml

```diff
@@ -0,0 +1,73 @@
+name: Bind notation
+description: |
+  Learn about command return types and how to bind the results.
+entities:
+  - name: Goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - |
+      Some command like are simple, like 'move: cmd ()', because
+      they do not have anything meaningful to return.
+    - |
+      But there are commands that do, like 'grab: cmd string'
+      that returns the name of the grabbed entity. Try it:
+    - |
+      'move; grab'
+    - |
+      To use the result you can use the bind notation:
+    - |
+      'move; t <- grab; place t'
+    - |
+      This can be useful if you for example do not care what you
+      grabbed and just want to move it to another cell.
+    - Once you are done, do 'place "Win"' to finish this challenge.
+    - You might need to 'grab' the entity you are standing on
+      or move to an empty cell.
```

Comment:
```suggestion
    - You might need to 'grab' the entity you are standing on
      or move to an empty cell first, since each cell can only
      contain at most one entity.
```